app/src/main/kotlin/cz/covid19cz/erouska/exposurenotifications/LocalNotificationsHelper.kt

```diff
@@ -53,7 +54,8 @@ object LocalNotificationsHelper {
         @StringRes title: Int,
         @StringRes text: Int,
         channelId: String,
-        context: Context?
+        context: Context?,
+        autoCancel : Boolean = false
```

Comment:
Default bych dal true, většinou chceme aby se to zrušilo po kliknutí, nezrušení je výjimka

Reply:
právě, že máme 3 typy notifikací a u dvou nechceme autoCancel :)

Reply:
Kdy nechceme autocancel? Napadá mě třeba když je eRouška pauznutá, tak bych taky chtěl autocancel - když na ní kliknu a zapnu eRoušku, tak tam notifikace nemá zůstat.